packages/react-tweet/src/twitter-theme/tweet-card.tsx

```diff
@@ -0,0 +1,104 @@
+import clsx from 'clsx'
+import {
+  BindingValues,
+  Card,
+  PollChoice,
+  StringValue,
+} from '../api/types/card.js'
+import s from './tweet-card.module.css'
+import infoStyles from './tweet-info.module.css'
+import { formatRemainingTime } from '../date-utils.js'
+import { formatVoteCount } from '../utils.js'
+
+export const TweetCard = ({ card }: { card: Card }) => {
+  // Poll card, e.g. https://x.com/elonmusk/status/1604617643973124097
+  return card.name.startsWith('poll') ? (
+    <TweetPoll poll={card.binding_values} />
+  ) : undefined
```

Comment:
I'm not sure if other type of cards exists yet.

Reply:
name: 

`poll2choice_text_only`
`poll3choice_text_only`
`poll4choice_text_only`